src/style.css

```diff
@@ -1 +1,61 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.form__input {
+  box-sizing: border-box;
+  display: flex;
```

Comment:
Avoid using tag names for styling. Instead of using 'display: flex;' directly on '.form__input', consider using a class to apply this style.